content/en/docs/installation/customization.md

```diff
@@ -372,6 +372,7 @@ The following flags can be used to control the advanced behavior of the various
 | `enableConfigMapCaching` (ABR) | true | Enables the ConfigMap caching feature. |
 | `enableDeferredLoading` (A) | true | Enables deferred (lazy) loading of variables (1.10.1+). Set to `false` to disable deferred loading of variables which was the default behavior in versions < 1.10.0. |
 | `enablePolicyException` (ABR) | true | Set to `true` to enable the [PolicyException capability](../writing-policies/exceptions.md). |
+| `enableReporting` (ABCR) | validate,mutate,mutateExisting,generate,imageVerify | Comma separated list to enables reporting for different rule types. (validate,mutate,mutateExisting,generate,imageVerify) |
```

Comment:
Can you clarify the default behavior?

Reply:
the default value field shows that all the values are enabled, should I add it to the description as well?

Reply:
Ah yes, that should be good.